cmd/utils.go

```diff
@@ -314,6 +315,17 @@ func UTCNow() time.Time {
 	return time.Now().UTC()
 }
 
+// UTCNowNTP - is similar in functionality to UTCNow()
+// but only used when we do not wish to rely on system
+// time.
+func UTCNowNTP() (time.Time, error) {
+	// ntp server is disabled
+	if ntpServer == "" {
+		return UTCNow(), nil
+	}
+	return ntp.Time(ntpServer)
```

Comment:
There is a default timeout of 5 seconds and NTP is using UPD, so if it unable to get a reply that will always add 5 seconds to the operation.

Enabling it by default seems to just be asking for trouble. I'd propose it to be opt-in, so users will also explicitly have to find a close-by NTP server.

Reply:
This is a proposed PR not a final one @klauspost

Reply:
Made it opt-in anyways @klauspost